packages/ember-data/lib/system/model/model.js

```diff
@@ -683,10 +683,7 @@ var Model = Ember.Object.extend(Ember.Evented, {
     successfully or rejected if the adapter returns with an error.
   */
   save: function() {
-    var model = this;
-    return this.reference.save().then(function() {
-      return model;
-    });
+    return this.reference.save();
```

Comment:
Sounds very reasonable. This also means bye bye PromiseObject from `system/reference.js` :+1: